tests/src/FunctionalJavascript/MessageFormTest.php

```diff
@@ -41,6 +41,7 @@ class MessageFormTest extends WebDriverTestBase {
     'contact',
     'contact_storage',
     'oe_contact_forms',
+    'oe_contact_forms_honeypot',
```

Comment:
This test is for the base module functionality. We need to write another test to check that honeypot is added correctly.